monitoring/uss_qualifier/webapp/docker-compose-webapp.yaml

```diff
@@ -0,0 +1,86 @@
+version: '3.9'
+services:
+  mock-riddp:
+    build:
+      context: ../../
+      dockerfile: mock_uss/Dockerfile
+    environment:
+      MOCK_USS_AUTH_SPEC: DummyOAuth(http://host.docker.internal:8085/token,uss1)
+      MOCK_USS_DSS_URL: http://host.docker.internal:8082
+      MOCK_USS_PUBLIC_KEY: /var/test-certs/auth2.pem
+      MOCK_USS_TOKEN_AUDIENCE: localhost,host.docker.internal
+      MOCK_USS_SERVICES: riddp
+    ports:
+      - 8073:5000
+    volumes:
+      - $PWD/build/test-certs:/var/test-certs:ro
+    command: gunicorn --preload --workers=1 --bind=0.0.0.0:5000 monitoring.mock_uss:webapp
+  mock-ridsp:
+    build:
+      context: ../../
+      dockerfile: mock_uss/Dockerfile
+    environment:
+      MOCK_USS_AUTH_SPEC: DummyOAuth(http://host.docker.internal:8085/token,uss1)
+      MOCK_USS_DSS_URL: http://host.docker.internal:8082
+      MOCK_USS_PUBLIC_KEY: /var/test-certs/auth2.pem
+      MOCK_USS_TOKEN_AUDIENCE: localhost,host.docker.internal
+      MOCK_USS_SERVICES: ridsp
+    ports:
+      - 8071:5000
+    volumes:
+      - $PWD/build/test-certs:/var/test-certs:ro
+    command: gunicorn --preload --workers=1 --bind=0.0.0.0:5000 monitoring.mock_uss:webapp
+  mock-scdsc:
+    build:
+      context: ../../
+      dockerfile: mock_uss/Dockerfile
+    environment:
+      MOCK_USS_AUTH_SPEC: DummyOAuth(http://host.docker.internal:8085/token,uss1)
+      MOCK_USS_DSS_URL: http://host.docker.internal:8082
+      MOCK_USS_PUBLIC_KEY: /var/test-certs/auth2.pem
+      MOCK_USS_TOKEN_AUDIENCE: localhost,host.docker.internal
+      MOCK_USS_SERVICES: scdsc
+    ports:
+      - 8074:5000
+    volumes:
+      - $PWD/build/test-certs:/var/test-certs:ro
+    command: gunicorn --preload --workers=1 --bind=0.0.0.0:5000 monitoring.mock_uss:webapp
+  redis:
+    image: redis:3-alpine
+    ports:
+      - 6379:6379
+  uss-host:
+    build:
+      context: ../../
+      dockerfile: Dockerfile
+    environment:
+      MOCK_HOST_USS_QUALIFIER_AUTH_SPEC: DummyOAuth(http://host.docker.internal:8085/token,uss1)
+      MOCK_HOST_USS_QUALIFIER_DSS_URL: http://host.docker.internal:8082
+      MOCK_HOST_USS_QUALIFIER_HOST_URL: http://localhost:8072
+      MOCK_HOST_USS_QUALIFIER_HOST_PORT: 8072
+      MOCK_HOST_USS_QUALIFIER_REDIS_URL: redis://redis-server:6379/0
+      MOCK_USS_TOKEN_AUDIENCE: localhost,host.docker.internal
+    ports:
+      - 8072:5000
+    volumes:
+      - $PWD/build/test-certs:/var/test-certs:ro
+      - ./tmp/uss-host-input-files:/mnt/app/input-files
+    links:
+      - redis:redis-server
+    command: gunicorn --preload --workers=2 --bind=0.0.0.0:5000 monitoring.uss_qualifier.webapp:webapp
+  rq-worker:
```

Comment:
rq-worker failing to start with following error, even though I have added `MOCK_USS_TOKEN_AUDIENCE`:
`rq-worker_1   | urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=8070): Max retries exceeded with url: /sp/uss2/tests/4dc152a3-dc59-47bb-b642-97316053ac98 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb41836d280>: Failed to establish a new connection: [Errno 111] Connection refused'))`